deployment/terraform/create.go

```diff
@@ -539,6 +539,7 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 	cfg.ServiceSettings.EnableLinkPreviews = model.NewBool(true)
 	cfg.ServiceSettings.EnablePermalinkPreviews = model.NewBool(true)
 	cfg.ServiceSettings.PostPriority = model.NewBool(true)
+	cfg.ServiceSettings.AllowCorsFrom = model.NewString("*")
```

Comment:
This obviously works and it's a quick fix. I suppose the more elaborate and security friendly way would be to add a URL using public dns/ip from either the proxy or the first app instance (if running a single app deployment). A potential further improvement I suppose.

For now, would it make sense to leave a comment explaining why we are setting this?

Reply:
Yep, it's a quick fix of course. A comment sounds fine.